test/NavbarSpec.js

```diff
@@ -36,4 +36,9 @@ describe('<Navbar />', () => {
     wrapper = shallow(<Navbar fixed />);
     expect(wrapper.hasClass('navbar-fixed')).to.eq(true);
   });
+
+  it('can have container', () => {
+    wrapper = shallow(<Navbar container />);
+    expect(wrapper.find('nav-wrapper container')).to.have.length(0);
```

Comment:
This should be have length 1 right? you can also use enzyme's `hasClass`

Reply:
`hasClass` here doesnt work.

`it('can have container', () => {
    wrapper = shallow(<Navbar container />);
    expect(wrapper.hasClass('container')).to.eq(true);
  });`

this returns `false` when `true` is expected

Reply:
has doesnt work if I use `mount`